cmd/operator-sdk/alpha/scorecard/cmd.go

```diff
@@ -38,18 +44,39 @@ func NewCmd() *cobra.Command {
 		RunE: func(cmd *cobra.Command, args []string) error {
 
 			var err error
-			o := scorecard.Options{}
+			o := scorecard.Options{
+				ServiceAccount: serviceAccount,
+				Namespace:      namespace,
+				List:           list,
+				BundlePath:     bundle,
+				OutputFormat:   output,
+				Cleanup:        cleanup,
+				WaitTime:       waitTime,
+			}
+			o.Client, err = scorecard.GetKubeClient(kubeconfig)
+			if err != nil {
+				return fmt.Errorf("could not get Kube connection %s", err.Error())
+			}
 			o.Config, err = scorecard.LoadConfig(config)
```

Comment:
I was going to punt that decision until we do the image bundle PR was my plan.  We need to resolve where in the bundle the scorecard config + other manifests (e.g. kuttl) will live in that
structure, figured it would be in a different PR than this one just to move this one along for now.